src/main/java/underdogs/devbie/recommendation/controller/AnswerRecommendationController.java

```diff
@@ -0,0 +1,69 @@
+package underdogs.devbie.recommendation.controller;
+
+import java.net.URI;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import lombok.AllArgsConstructor;
+import underdogs.devbie.auth.controller.resolver.LoginUser;
+import underdogs.devbie.recommendation.dto.RecommendationRequest;
+import underdogs.devbie.recommendation.dto.RecommendationResponse;
+import underdogs.devbie.recommendation.service.AnswerRecommendationService;
+import underdogs.devbie.user.domain.User;
+
+@Controller
```

Comment:
@RestController 를 사용해야하지 않을까요~~? 

```suggestion
@RestController
```